docs/zh/latest/admin-api.md

```diff
@@ -65,6 +65,46 @@ deployment:
             port: 9180                  # Admin API 监听的 端口，必须使用与 node_listen 不同的端口。
 ```
 
+### 使用环境变量 {$using-environment-variables}
+
+要通过环境变量进行配置，可以使用 `${{VAR}}` 语法。例如：
+
+```yaml title="./conf/config.yaml"
+deployment:
+  admin:
+    admin_key:
+    - name: admin
+      key: ${{ADMIN_KEY}}
+      role: admin
+    allow_admin:
+    - 127.0.0.0/24
+    admin_listen:
+      ip: 0.0.0.0
+      port: 9180
+```
+
+然后在 `make init` 之前运行 `export ADMIN_KEY=$your_admin_key`.
```

Comment:
🤔 So it means if users use APISIX with env, they have to use make init?

Reply:
It has little to do with environment variables. `make init` is mainly used to verify whether the configuration is valid and update nginx. conf, and update the data configuration of etcd.

Reply:
It has little to do with environment variables. make init is mainly used to verify whether the configuration is valid and update nginx. conf, and update the data configuration of etcd.

Reply:
Ping @juzhiyuan .